index.ts

```diff
@@ -1,8 +1,8 @@
-import * as Promise from 'bluebird';
-import { writeFile } from 'fs';
+import Bluebird from 'bluebird';
+import { writeFileSync } from 'fs';
 
-const writeFileAsync = Promise.promisify(writeFile);
-Promise.config({
+const writeFile = Bluebird.promisify(writeFileSync);
```

Comment:
This needs to remain using the `writeFile` version as the sync one is, well, synchronous. If you're going to use the sync version you'd be better off not using promises at all (but it'd overall be a very bad idea)
```suggestion
const writeFileAsync = Bluebird.promisify(writeFile);
```